src/adapter/adapter.js

```diff
@@ -158,6 +158,9 @@ let YouboraAdapter = youbora.Adapter.extend({
 
   /** @returns {void} - Listener for 'playing' event. */
   playingListener: function () {
+    if (this.player.drmInfo().scheme) {
+      this.plugin.options['content.drm'] = this.player.drmInfo().scheme;
```

Comment:
need to think if we want to be aligned with android values. see https://github.com/kaltura/playkit-android-youbora/pull/81

Reply:
aligned with android
https://github.com/kaltura/playkit-android/blob/dev/playkit/src/main/java/com/kaltura/playkit/PKDrmParams.java